web/react/components/textbox.jsx

```diff
@@ -335,13 +335,17 @@ export default class Textbox extends React.Component {
                     dangerouslySetInnerHTML={{__html: this.state.preview ? TextFormatting.formatText(this.props.messageText) : ''}}
                 >
                 </div>
-                <a
-                    style={{visibility: previewLinkVisible ? 'visible' : 'hidden'}}
-                    onClick={this.showPreview}
-                    className='textbox-preview-link'
-                >
-                    {this.state.preview ? 'Edit message' : 'Preview'}
-                </a>
+                {
+                    Utils.featureToggle('#1389') ?
```

Comment:
@florianorben maybe move this out of the return statement and use the full-length if statement?

Kind of like this:
```
let previewLink = null;
if (Utils.featureToggle('#1389')) {
    previewLink  = // stuff
}
```